client/src/components/Header/Header.js

```diff
@@ -13,7 +13,9 @@ class Header extends React.Component {
     links = [
         {name: 'Home', route: '/', exact: true},
         {name: 'Map', route: '/map'},
-        {name: 'About', route: '/about'}
+        {name: 'About', route: '/about'},
+        {name: 'Create', route: '/create', exact: true, protected: true},
+        {name: 'Bulk Create', route: '/bulk-create', exact: true, protected: true}
```

Comment:
1. How does this look on small monitor sizes? I don't think there's actually room for this...
2. For Create we can just update the suggest a change button to link to it instead

Reply:
I remove the Create link from the Header and moved it to the Suggest a Change Button on the Homepage.

It doesn't look awful on a smaller monitor, the Log Out and My Account buttons kind of slide off the side though.